Tests/SourceKitLSPTests/SwiftPMIntegration.swift

```diff
@@ -56,4 +56,88 @@ final class SwiftPMIntegrationTests: XCTestCase {
         deprecated: false),
     ])
   }
+
+  func testAddFile() throws {
```

Comment:
Need to update Linux tests.  Incidentally, I wonder if we could switch to automatic test discovery on Linux at this point - it's been around for a while now.

Reply:
Thanks for catching this. I’ll try to switch to automatic test discovery in a follow-up PR.